homeassistant/components/samsungtv/const.py

```diff
@@ -6,6 +6,18 @@
 
 DEFAULT_NAME = "Samsung TV"
 
+CONFIG_NAME = "HomeAssistant"
+CONFIG_ID = "ha.component.samsung"
+
+CONF_DESCRIPTION = "description"
 CONF_MANUFACTURER = "manufacturer"
 CONF_MODEL = "model"
 CONF_ON_ACTION = "turn_on_action"
```

Comment:
This is a mixture between config names and config values which is hard to distinguish.

Reply:
Rename  in CONFIG_VALUE_